src/dotnet/APIView/APIView/Model/CodeFileTokenKind.cs

```diff
@@ -31,6 +31,8 @@ public enum CodeFileTokenKind
         TableColumnName = 24,
         TableCellBegin = 25,
         TableCellEnd = 26,
-        LeafSectionPlaceholder = 27
+        LeafSectionPlaceholder = 27,
+        ExternalLinkStart = 28,
```

Comment:
I don't think we need two tokens here. Just one token type to show value is external link is good enough. Start and end are required when we need to include too many tokens wrapped within.

Reply:
@praveenkuttappan the original issue stated that 

> Building off the previous, adding a (link) tag to certain lines that would provide a direct link to the source code line in the relevant PR.

Which I believe would require two tokens as any line could contain multiple tokens (correct me if I'm wrong).

I also believe that the separate tokens allows us to have links with a separate hyperlink and text (again, correct me if I'm wrong).

Happy to change it to one token though, let me know your thoughts.

Reply:
I do think there is some value to the approach that @Vailorium has proposed in this PR. It means that we are not bound to show potentially long hyperlinks when that it the required hyperlink (nor do we have to manually shorten these long URLs so as to not invade our API reviews), and we can then instead shorten them to some link text, in the same way that web browsers enable this. In other words, it allows for us to separate the 'a href=..' from the link text itself.